client/src/panels/panel_declarations/results/results_common.js

```diff
@@ -185,6 +186,16 @@ const filter_and_genericize_doc_counts = (counts, doc_key) => {
   return doc_counts_with_generic_keys;
 };
 
+const result_color_scale = d3.scaleOrdinal()
+  .domain(["met","not_met","not_available","future"])
+  .range([
+    newIBCategoryColors[0],
+    newIBCategoryColors[1],
+    newIBCategoryColors[2],
+    newIBCategoryColors[3],
+  ]);
```

Comment:
Why not just an object mapping keys to colours? What's the d3 wrapping add?

Reply:
For nivo?